arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -2868,6 +2868,18 @@ void RocksDBEngine::scheduleFullIndexRefill(std::string const& database,
   f.scheduleFullIndexRefill(database, collection, iid);
 }
 
+bool RocksDBEngine::autoRefillIndexCaches() const {
```

Comment:
NIT 
maybe better to get user access to the RocksDBIndexCacheRefillFeature& instead of two methods?

Reply:
I intentionally didn't want storage engine-agnostic code (such as transaction/Methods.cpp) to access RocksDB-specific code/classes.
That's why I generalized the access via virtual functions in the generic storage engine interface.

Reply:
Ok